docs/suspensive.org/src/pages/docs/react/ErrorBoundary.en.mdx

```diff
@@ -364,6 +365,226 @@ export const ErrorAfter2s = () => {
+import { CustomErrorAfter2s } from './CustomErrorAfter2s'
+
+export const Example = () => {
+  return (
+    <ErrorBoundary fallback={({ error }) => <>{error.message} of Parent</>}>
+      <ErrorBoundary
+        shouldCatch={CustomError}
+        fallback={({ error }) => <>{error.message} of Child</>}
+      >
+        <CustomErrorAfter2s />
+      </ErrorBoundary>
+    </ErrorBoundary>
+  )
+}
+```
+
+  </Tabs.Tab>
+  <Tabs.Tab>
+  
+```tsx {10}
+import { ErrorBoundary } from '@suspensive/react'
+import { createElement } from 'react'
+import { CustomError } from './CustomError'
+import { CustomErrorAfter2s } from './CustomErrorAfter2s'
+
+export const Example = () => {
+  return (
+    <ErrorBoundary fallback={({ error }) => <>{error.message} of Parent</>}>
+      <ErrorBoundary
+        shouldCatch={(error) => error instanceof CustomError}
+        fallback={({ error }) => <>{error.message} of Child</>}
+      >
+        <CustomErrorAfter2s />
+      </ErrorBoundary>
+    </ErrorBoundary>
+  )
+}
+```
+  </Tabs.Tab>
+</Tabs>
+
+You can also apply multiple criteria through array.
+
+```tsx {10}
+import { ErrorBoundary } from '@suspensive/react'
+import { createElement } from 'react'
+import { CustomError } from './CustomError'
+import { CustomErrorAfter2s } from './CustomErrorAfter2s'
+
+export const Example = () => {
+return (
+  <ErrorBoundary fallback={({ error }) => <>{error.message} of Parent</>}>
+    <ErrorBoundary
+      shouldCatch={[false, CustomError, (error) => error instanceof CustomError]}
+      fallback={({ error }) => <>{error.message} of Child</>}
+    >
+      <CustomErrorAfter2s />
+    </ErrorBoundary>
+  </ErrorBoundary>
+  )
+}
+```
+
+<Sandpack>
+
+```tsx Example.tsx active
+import { ErrorBoundary } from '@suspensive/react'
+import { createElement } from 'react'
+import { CustomError } from './CustomError'
+import { CustomErrorAfter2s } from './CustomErrorAfter2s'
+
+export const Example = () => {
+  return (
+    <ErrorBoundary fallback={({ error }) => <>{error.message} of Parent</>}>
+      <ErrorBoundary
+        shouldCatch={CustomError}
+        fallback={({ error }) => <>{error.message} of Child</>}
+      >
+        <CustomErrorAfter2s />
+      </ErrorBoundary>
+    </ErrorBoundary>
+  )
+}
+```
+
+```tsx CustomError.ts
+
+export class CustomError extends Error {
+  constructor(...args: ConstructorParameters<ErrorConstructor>) {
+    super(...args)
+    console.error(...args)
+  }
+}
+```
+
+```tsx CustomErrorAfter2s.tsx
+import { useState, useEffect } from "react";
+import { CustomError } from "./CustomError";
+
+export const CustomErrorAfter2s = () => {
+  const [asyncState, setAsyncState] = useState<
+    { isError: true; error: CustomError } | { isError: false; error: null }
+  >({
+    isError: false,
+    error: null
+  });
+
+  useEffect(() => {
+    setTimeout(() => {
+      setAsyncState({
+        isError: true,
+        error: () => new CustomError("unauthorized")
```

Comment:
Could you remove all unnecessary context in this doc please? In my opinion, text "unauthorized" could make misunderstanding concept shouldCatch

```suggestion
        error: () => new CustomError("⚠️ Error made by CustomError Constructor")
```

Reply:
I agree, I changed it!